### DISCUSSION TITLE
cs-knowledge-graph-3bb3c2f415e14760a50be60aa0792b5f

### DISCUSSION THREAD

**Discussion by giscus** [wustep/me#4] [Posts]
# cs-knowledge-graph-3bb3c2f415e14760a50be60aa0792b5f

This article explores some of the different paths and endless breadth and depth of a software development career or a computer science degree. Its target audience is mostly students and early-in-career folks. 

http://localhost:3000/cs-knowledge-graph-3bb3c2f415e14760a50be60aa0792b5f

**Comment by wustep** [comment]
tester!

**Reply by wustep** [reply]
asdf

**Reply by wustep** [reply]
testasdf

**Comment by wustep** [comment]
hello world!

**Reply by wustep** [reply]
testerasf

**Comment by wustep** [comment]
i love it

**Comment by wustep** [comment]
test

**Reply by wustep** [reply]
asfzxcv

**Comment by wustep** [comment]
# hello world!
